### PR TITLE
add Prerequisite and Gatsby drawback on `why-gatsby` page

### PR DESCRIPTION
**Pull request by mildronize**
I've reviewed `why-gatsby` page. I think something is missing. I've decided to add the drawback to select Gatsby as the web framework tools. It might be beneficial for who are making the decision. As you can see in the screenshot below:

Screen Shot 2564-04-21 at 07 03 24

Any suggestion is welcome, please review my pull request 😊

### CODE REVIEW

File: content/web-frameworks/gatsby/why-gatsby.md [hunk 1]
```diff
@@ -31,6 +35,17 @@ showToc: true
   - [GitHub Action workflow for GitHub Pages](/programming-cookbook/github-actions/deploy-to-github-pages)
   - [GitHub Action workflow for CloudFlare Workers](/programming-cookbook/github-actions/deploy-to-cloudflare-workers)
 
+# Gatsby don't be suitable if:
```
Comment (aaronamm) [inline content/web-frameworks/gatsby/why-gatsby.md]: Very minor, suitable is adjective. Then we use it with `is`.

`Gatsby isn't suitable if:`

File: content/web-frameworks/gatsby/why-gatsby.md [hunk 2]
```diff
@@ -31,6 +35,17 @@ showToc: true
...
+
+- The web admin is not a developer. To modify the content or design, it requires programming skill.
+- You want admin control panel to modify the content.
+- Your content will be changed quite often, it'll make difficult to modify the content when compare to use admin panel.
+  - If you want to update the content in mobile or public computer, it might not convenience. The computer requires specify tools such as an editor, git command etc.
+- You're sticky on the traditional technology like basic html and css renderer tools. Other tools will be suitable for you such as:
```
Comment (aaronamm) [inline content/web-frameworks/gatsby/why-gatsby.md]: Very minor stick is a verb, think we should say You stick to or change you You rely on.
Please also use CAPITAL for abbreviation, HTML, CSS.

**Review by aaronamm — APPROVED** [review]

**Comment by aaronamm** [comment]
@mildronize  Thanks for adding a useful content. Please fix PR feedbacks and I will merge it.

Thanks.

File: content/web-frameworks/gatsby/why-gatsby.md [hunk 3]
```diff
@@ -31,6 +35,17 @@ showToc: true
   - [GitHub Action workflow for GitHub Pages](/programming-cookbook/github-actions/deploy-to-github-pages)
   - [GitHub Action workflow for CloudFlare Workers](/programming-cookbook/github-actions/deploy-to-cloudflare-workers)
 
+# Gatsby don't be suitable if:
+
+- The web admin is not a developer. To modify the content or design, it requires programming skill.
+- You want admin control panel to modify the content.
+- Your content will be changed quite often, it'll make difficult to modify the content when compare to use admin panel.
```
Comment (surrealist) [inline content/web-frameworks/gatsby/why-gatsby.md]: You use the word _admin control panel_ from previous bullet. It better to use consistent word. For example:

```
..., it's more difficult to modify content than using admin control panel.
```

File: content/web-frameworks/gatsby/why-gatsby.md [hunk 4]
```diff
@@ -31,6 +35,17 @@ showToc: true
...
+  - If you want to update the content in mobile or public computer, it might not convenience. The computer requires specify tools such as an editor, git command etc.
```
Comment (surrealist) [inline content/web-frameworks/gatsby/why-gatsby.md]: I'm not sure you indent this bullet intentionally?

File: content/web-frameworks/gatsby/why-gatsby.md (hunk 1 above)
Comment (mildronize) [reply content/web-frameworks/gatsby/why-gatsby.md]: Oh, it's my fault. Thank a lot.

File: content/web-frameworks/gatsby/why-gatsby.md (hunk 2 above)
Comment (mildronize) [reply content/web-frameworks/gatsby/why-gatsby.md]: Thanks, I'll fix it.

File: content/web-frameworks/gatsby/why-gatsby.md (hunk 4 above)
Comment (mildronize) [reply content/web-frameworks/gatsby/why-gatsby.md]: Yes, I think this is come with same context of the parent bullet.

File: content/web-frameworks/gatsby/why-gatsby.md (hunk 3 above)
Comment (mildronize) [reply content/web-frameworks/gatsby/why-gatsby.md]: Oh, thanks a lot. Your eye is sharpen.  😊

**Comment by mildronize** [comment]
@aaronamm @surrealist I've fixed following your suggestion and reviewed the grammar. Thank you a lot.  😊

File: content/web-frameworks/gatsby/why-gatsby.md (hunk 1 above)
Comment (surrealist) [reply content/web-frameworks/gatsby/why-gatsby.md]: Did you fix this?

Comment (mildronize) [reply content/web-frameworks/gatsby/why-gatsby.md]: Yes, thanks

**Comment by aaronamm** [comment]
@mildronize 

Thank you so much.

**Pull request merged**